packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/Advanced/WalletPassword/Settings/index.js

```diff
@@ -25,11 +25,19 @@ class SettingsContainer extends React.PureComponent {
     this.props.updateUI({ updateToggled: !this.props.ui.updateToggled })
   }
 
+  validateCurrentPassword = (value) =>
+    value === this.props.currentWalletPassword ? undefined : 'Incorrect password'
+
+  validatePasswordConfirmation = (value, allValues) =>
+    (value === allValues.newPassword) ? undefined : 'Passwords do not match'
+
   render () {
-    const { ui, ...rest } = this.props
+    const { ui, currentWalletPassword, ...rest } = this.props
```

Comment:
I don't think this is used?

Reply:
It's to remove it from being passed onwards